CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Help/Open with tutorial/OpenWithTutorialCollectionCell.swift

```diff
@@ -9,130 +9,131 @@
     }()
-    
+
     lazy var stepIndicatorCircle: UIView = {
         var view = UIView(frame: .zero)
         view.translatesAutoresizingMaskIntoConstraints = false
         view.frame.size = self.stepIndicatorCircleSize
-        
+
         updateStepIndicatorCircleColor(stepIndicatorCircle: view)
-        
+
         view.layer.borderWidth = 1
         view.layer.cornerRadius = self.stepIndicatorCircleSize.width / 2
         return view
     }()
-    
+
     private func updateStepIndicatorCircleColor(stepIndicatorCircle: UIView) {
-        
-        stepIndicatorCircle.layer.borderColor = UIColor.from(giniColor: GiniConfiguration.shared.indicatorCircleColor).cgColor
+
+        stepIndicatorCircle.layer.borderColor = UIColor.from(
+            giniColor: GiniConfiguration.shared.indicatorCircleColor).cgColor
     }
-    
+
     lazy var stepTitle: UILabel = {
         var label = UILabel()
         label.numberOfLines = 0
         label.translatesAutoresizingMaskIntoConstraints = false
-        
+
         return label
     }()
-    
+
     lazy var stepSubTitle: UILabel = {
         var label = UILabel()
         label.numberOfLines = 0
         label.translatesAutoresizingMaskIntoConstraints = false
-        
+
         if #available(iOS 13.0, *) {
             label.textColor = .secondaryLabel
         } else {
             label.textColor = .lightGray
         }
-        
+
         return label
     }()
-    
+
     lazy var stepImage: UIImageView = {
         var imageView = UIImageView()
         imageView.translatesAutoresizingMaskIntoConstraints = false
         imageView.contentMode = .scaleAspectFit
         imageView.layer.shadowOffset = CGSize(width: 0, height: 4)
-        
+
         updateStepImageShadow(stepImage: imageView)
-        
+
         imageView.layer.shadowOpacity = 0.1
         imageView.layer.shadowRadius = 14
         imageView.layer.shadowPath = UIBezierPath(rect: imageView.bounds).cgPath
         return imageView
     }()
-    
+
     private func updateStepImageShadow(stepImage: UIImageView) {
-        
+
         if #available(iOS 13.0, *) {
             stepImage.layer.shadowColor = Colors.Gini.shadowColor.cgColor
         } else {
             stepImage.layer.shadowColor = UIColor.black.cgColor
         }
     }
-    
+
     override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-        
+
         super.traitCollectionDidChange(previousTraitCollection)
-        
+
         if #available(iOS 13.0, *) {
             if traitCollection.hasDifferentColorAppearance(comparedTo: previousTraitCollection) {
-                
+
                 updateStepIndicatorCircleColor(stepIndicatorCircle: stepIndicatorCircle)
                 updateStepImageShadow(stepImage: stepImage)
             }
         }
     }
-    
+
     override init(frame: CGRect) {
         super.init(frame: frame)
-        
+
         if #available(iOS 13.0, *) {
             backgroundColor = .systemBackground
```

Comment:
@chriskryn  could you please add this to our palette as well
and use UIColorPreferred😉